exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -157,7 +157,7 @@ func populateOtherDimensions(attributes pcommon.Map, span *Span) {
 			if err == nil && statusString != 0 {
 				statusInt = int64(statusString)
 			}
-			if statusInt >= 400 {
+			if statusInt >= 500 || (statusInt >= 400 && span.Kind != 2) {
```

Comment:
There is a gRPC as well. Please update that.

https://github.com/SigNoz/signoz-otel-collector/blob/8decaf03afdfcd7d7167738642972f2f2afc7f45/exporter/clickhousetracesexporter/clickhouse_exporter.go#L198-L207

Reply:
For gRPC apparently it's not very clear which are server side errors.
Go through these docs, you will see that most status codes are shared among client and server: https://grpc.github.io/grpc/core/md_doc_statuscodes.html